spock-specs/src/test/groovy/org/spockframework/smoke/parameterization/MethodParameters.groovy

```diff
@@ -263,6 +263,7 @@ def foo() {
         assert it.arguments.size() == 1
         assert it.arguments.first() == MISSING_ARGUMENT
         it.arguments[0] = 'foo'
+        it.proceed()
```

Comment:
Good catch! And probably it is enough as probably we don't expect to have that line removed for any reason again. However, what do you think about something like:

```
    when:
    runner.runSpecBody """
@Shared
boolean methodExecuted = false

@Foo
def $fixtureMethod(x) {
  assert x == 'foo'
  methodExecuted = true
}

def foo() {
  expect:
  methodExecuted
}
"""
```

to verify that the fixtureMethod was in fact executed?

Reply:
I don't think that is a good idea or necessary.
I always recommend "don't trust a test you didn't see fail for the correct reason", but there I obviously didn't follow my own words as I forgot the `proceed` right from the start.
That's unfortunately a typical error of mine.

If we would add such a check here, we should add it to any tests with extensions.
And actually we need to verify each single implicit assertion, to be consistent,
because someone could move it into a block and thus void the implicit assertion. :-)